docs/configure/webpack.md

```diff
@@ -91,6 +91,8 @@ If you're using a non-standard Storybook config directory, you should put `main.
 
 If you have an existing webpack config for your project and want to reuse this app's configuration, you can import your main webpack config into Storybook's [`.storybook/main.js`](./overview.md#configure-story-rendering) and merge both:
 
+NB! If you are using Vue CLI you need to use the resolved webpack config file, which is found at `<projectRoot>/node_modules/@vue/cli-service/webpack.config.js`.
+
```

Comment:
@AasmundEndresen we have a little bit of room for improvement here, instead of leaving it here, we can change this into the following:
```html
<div class="aside">
💡 <strong>Note:</strong> Projects initialized via generator (e.g, Vue CLI) may require that you import their own webpack config file (i.e., <code>/projectRoot/node_modules/@vue/cli-service/webpack.config.js</code>) in order to use a certain feature with Storybook. For other generators, make sure to check the documentation for instructions. 
</div>
```

And put this aside after the code snippet below. 

With this, we keep the documentation framework-agnostic and address your use case.

Let me know if you're ok with this change and we'll go from there.